types/storybook-addon-intl/logo.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '*.png'
+declare module '*.jpg'
```

Comment:
Is this file still needed after the change at c887eb5676471238cee033ea1ac48a2c16220bab?

Reply:
No, we have removed it :)